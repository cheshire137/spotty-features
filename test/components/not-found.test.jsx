import React from 'react'
import TestUtils from 'react-addons-test-utils'
import renderer from 'react-test-renderer'

import NotFound from '../../src/components/not-found.jsx'

describe('NotFound', () => {
  test('matches snapshot', () => {
    const component = renderer.create(<NotFound />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

  test('renders', () => {
    const renderer = TestUtils.createRenderer()
    renderer.render(<NotFound />)

    const result = renderer.getRenderOutput()
    expect(result.props.children).toBe('404 Not Found')
    expect(result.type).toBe('h1')
  })
})
