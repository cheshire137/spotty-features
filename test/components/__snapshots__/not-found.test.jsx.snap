exports[`NotFound matches snapshot 1`] = `
<h1
  className="title">
  404 Not Found
</h1>
`;
